README.md

```diff
@@ -121,6 +121,13 @@ See [Rules.md](doc/Rules.md) for more details.
 > `header` aliases (e.g. `heading-increment` is also available as `header-increment`).
 > The use of `header` is deprecated and provided for backward-compatibility.
 
+### Custom rules
```

Comment:
This new section will probably fail a CI test

Reply:
Updated markdownlint-test.js to add new heading.